recipes/graylog-datanode/patches/datanode.conf.patch

```diff
@@ -19,6 +19,7 @@
 -# opensearch_data_location = data
 -# opensearch_logs_location = logs
 +opensearch_location = /usr/share/graylog-datanode/dist
++opensearch_plugins_location = /usr/share/graylog-datanode/dist/plugins
```

Comment:
This setting is no longer necessary, plugins are pre-installed now

Reply:
Thanks!